src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -279,7 +285,11 @@ def _must_gather_kube_api(cluster_name, cluster_deployment, agent_cluster_instal
 
 
 def get_cluster_events_path(cluster, output_folder):
-    return os.path.join(output_folder, f"cluster_{cluster['id']}_events.json")
+    return os.path.join(output_folder, f"cluster_{cluster['id']}_cluster_events.json")
```

Comment:
Need to change that. It already has a `cluster` prefix

Reply:
done